### PR TITLE
Fix list assignment

### PR DESCRIPTION
**Pull request by vgalaktionov**
This fixes issue #622.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ export function app(state, actions, view, container) {
         element[name][i] = value == null || value[i] == null ? "" : value[i]
       }
     } else {
-      if (typeof value === "function" || (name in element && !isSVG)) {
+      if (typeof value === "function" || (name in element && name !== 'list' && !isSVG)) {
```
Comment (JorgeBucaran) [inline src/index.js]: @vgalaktionov Please use double quotes → `"`.

**Comment by vgalaktionov** [comment]
Done! Also added a test, not sure if it's the right place but it looked sensible.

File: test/dom.test.js [hunk 2]
```diff
@@ -758,3 +758,13 @@ testTreeSegue("elements with falsey values", [
...
 ])
+
+testTreeSegue("input list attribute", [
+  {
+    tree: h("input", {
+      "list": "foobar"
+    }),
+    html: `<input list="foobar">`
+  }
+])
```
Comment (frenzzy) [inline test/dom.test.js]: Just curious, is this test fails without the change?

**Comment by JorgeBucaran** [comment]
Thanks, @vgalaktionov! 🎉

**Pull request merged**